spec/spec_helper.rb

```diff
@@ -18,7 +20,7 @@
 
   SimpleCov.formatters = [
     SimpleCov::Formatter::Console,
-    SimpleCov::Formatter::Codecov,
+    SimpleCov::Formatter::Codecov
```

Comment:
I'm not a fan of this... I prefer the trailing comma on all items in a list...

Reply:
yes! dito! we should enforce that via rubocop

Reply:
It is being enforced by rubocop... just not the "right" way...  I'll change it.